test/Sentry.Unity.Android.Tests/Sentry.Unity.Android.Tests.csproj

```diff
@@ -6,5 +6,6 @@
     <ProjectReference Include="../../src/sentry-dotnet/src/Sentry/Sentry.csproj" Private="false" />
     <ProjectReference Include="../../src/Sentry.Unity/Sentry.Unity.csproj" Private="false" />
     <ProjectReference Include="../../src/Sentry.Unity.Android/Sentry.Unity.Android.csproj" Private="false" />
+    <ProjectReference Include="../Sentry.Unity.Tests/Sentry.Unity.Tests.csproj" />
```

Comment:
I believe this can cause issues. In .NET IIRC this would return the tests twice. Does it work OK with Unity?
Often in this case we have a test helper class. e.g: 

https://github.com/getsentry/sentry-dotnet/tree/main/test/Sentry.Testing

Reply:
It does not seem to cause any issues in Unity. We do have those `SharedClasses` so we could even move it there.